### PR TITLE
Update setup-machine-arktos.sh

### PR DESCRIPTION
**Pull request by Hong-Chang**
setup-machine-arktos.sh is outdated. This pr is to update it by:
1. Remove unnecessary installs
2. Use go version go1.13.9
3. Stop using multi-tenant version of containerd

### CODE REVIEW

File: setup-machine-arktos.sh [hunk 1]
```diff
@@ -185,18 +134,6 @@ pip3 install PyYAML
 
 ####################
 
-echo Setup: Change Containerd
```
Comment (h-w-chen) [inline setup-machine-arktos.sh:188]: It seems that customized containerd with multi-tenant ability is still needed for arktos integration, for the current mizarcni implementation. You can have some explanation why the upstream containerd is used here as temporary placement.

**Review by h-w-chen — CHANGES_REQUESTED** [review]
please add comments about customized containerd. It is fine not to use it right now; it should have some comment as reminder/explanation why such a temporary measure is taken.

File: setup-machine-arktos.sh (hunk 1 above)
Comment (Hong-Chang) [reply setup-machine-arktos.sh:188]: I filed issue https://github.com/CentaurusInfra/mizar/issues/433 for tracking.

**Review by h-w-chen — APPROVED** [review]
lgtm

**Pull request merged**